bannzai/Pilll
# Implement in app purchase

## What
課金機能と課金導線の実装

## Links
resolved: https://github.com/bannzai/_Pilll/issues/332
resolved: https://github.com/bannzai/_Pilll/issues/273



## Testing
### Developer
Storekit configuration を設定していればXcodeからのビルドで擬似的なテストを行える

### QA
Non Developer向けの日本語の良記事があった
https://spinners.work/posts/ios-sandbox/

ざっくりとした流れ
TestFlight でアプリを配布 -> Sandbox Tester  を Apple Developersから作ってそれをiPhoneから設定する -> Test

Revenue Cat が公開してくれている Check lists があったのでこれも参考にする
https://docs.revenuecat.com/docs/launch-checklist
